src/main.rs

```diff
@@ -30,6 +30,13 @@ fn main() {
                 .help("levels adjustment to match Smash Ultimate")
                 .takes_value(false),
         )
+        .arg(
+            Arg::with_name("is_slim")
+                .short("i")
+                .long("i")
```

Comment:
The long version should be "slim". I would also delete the short version "i" to avoid confusion. This code will probably need to be completely reworked later.

Reply:
didn't even notice i forgot to set a long. oops